Source/AwsCommonRuntimeKit/io/retryer/RetryToken.swift

```diff
@@ -4,7 +4,7 @@
 import AwsCIo
 
 /// This is just a wrapper for aws_retry_token which user can not create themself but pass around once acquired.
-public class CRTAWSRetryToken {
+public class RetryToken {
```

Comment:
It might be good to have the `AWS` prefix, since it appears that this token is aws specific given the underlying C type is `aws_retry_token`?

Reply:
I could go either way here, so defer to your best judgement

Reply:
Thanks, I have prefixed it with `aws` as Retry Strategy is only used in Auth layer which is specific to AWS. Btw, In C, everything is prefixed with `aws`.